src/client.ts

```diff
@@ -32,6 +32,12 @@ export interface PredictOptions {
    * This performs a full OCR operation on the server and will increase response time.
    */
   allWords?: boolean;
+  /**
+   * Whether to include the full ocr text. Only available on compatible APIs.
+   *
+   * This performs a full OCR operation on the server and will increase response time.
```

Comment:
This is only true when use_text=false and use_vision=true.
If use_text is true, then the OCR is done regardless.

```suggestion
   * This performs a full OCR operation on the server and may increase response time on some APIs.
```